### PR TITLE
add formal test for CSA.

### PR DESCRIPTION
**Pull request by sequencer**

### CODE REVIEW

**Comment by sequencer** [comment]
mill is 0.9.7 now

**Comment by sequencer** [comment]
https://github.com/com-lihaoyi/mill#0-9-7-2021-05-14

**Review by ljwljwljwljw — APPROVED** [review]

File: build.sc [hunk 1]
```diff
@@ -43,7 +43,7 @@ class arithmetic extends ScalaModule with ScalafmtModule with PublishModule { m
   object tests extends Tests {
     override def ivyDeps = Agg(ivy"com.lihaoyi::utest:latest.integration")
 
-    def testFramework = "utest.runner.Framework"
+    def testFrameworks = Seq("utest.runner.Framework")
   }
```
Comment (sequencer) [inline build.sc:47]: Deprecated in https://github.com/com-lihaoyi/mill#0-9-7-2021-05-14
> TestModule: Added new testFramework target and only support one test framework. Deprecated testFrameworks targets.

Should revert here.

File: .mill-version [hunk 2]
```diff
@@ -0,0 +1 @@
+0.9.7
```
Comment (sequencer) [inline .mill-version:1]: I prefer not to lock version. Just use upstream up-to-date version

**Pull request merged**